### DISCUSSION TITLE
Fix translation errors

### DISCUSSION THREAD

**Discussion by maxbobse** [dbeaver/dbeaver#32987] [DBeaver Development] [answered]
Hi,
I just submitted 

Minor translation issue with German translation "Zeigen alle Datenbanken an" #32986
https://github.com/dbeaver/dbeaver/issues/32986

Happy to fix it myself in the translation file if someone could walk me through the steps required to make the pull request :)

Max

**Comment by E1izabeth** [comment, marked as answer]
Hope it will be helpful
https://github.com/dbeaver/dbeaver/wiki/Localization#contribution 
https://github.com/dbeaver/dbeaver/wiki/Contribute-your-code

**Reply by maxbobse** [reply]
Done - hope I did not break anything :D

**Reply by E1izabeth** [reply]
> hope I did not break anything :D

Well, we will check :D

**Reply by E1izabeth** [reply]
I don't see your Pull Request. Have you 1) committed and pushed your changes and then 2) created pull request to the forked repo?

**Reply by E1izabeth** [reply]
@maxbobse you can press "Create pull request" button here https://github.com/dbeaver/dbeaver/compare/devel...maxbobse:dbeaver:patch-1

**Reply by maxbobse** [reply]
Got it, thanks :)
